integration-cli/docker_cli_create_test.go

```diff
@@ -101,6 +101,94 @@ func TestCreateHostConfig(t *testing.T) {
 	logDone("create - hostconfig")
 }
 
+func TestCreateWithPortRange(t *testing.T) {
+	runCmd := exec.Command(dockerBinary, "create", "-p", "3300-3303:3300-3303/tcp", "busybox", "echo")
+	out, _, _, err := runCommandWithStdoutStderr(runCmd)
+	if err != nil {
+		t.Fatal(out, err)
+	}
+
+	cleanedContainerID := stripTrailingCharacters(out)
+
+	inspectCmd := exec.Command(dockerBinary, "inspect", cleanedContainerID)
+	out, _, err = runCommandWithOutput(inspectCmd)
+	if err != nil {
+		t.Fatalf("out should've been a container id: %s, %v", out, err)
+	}
+
+	containers := []struct {
+		HostConfig *struct {
+			PortBindings map[string][]struct {
+				HostIp   string
+				HostPort string
+			}
+		}
+	}{}
+	if err := json.Unmarshal([]byte(out), &containers); err != nil {
+		t.Fatalf("Error inspecting the container: %s", err)
+	}
+	if len(containers) != 1 {
+		t.Fatalf("Unexpected container count. Expected 0, received: %d", len(containers))
+	}
+
+	c := containers[0]
+	if c.HostConfig == nil {
+		t.Fatalf("Expected HostConfig, got none")
+	}
+
+	if len(c.HostConfig.PortBindings) != 4 {
+		t.Fatalf("Expected 4 ports bindings, got %d", len(c.HostConfig.PortBindings))
+	}
+
+	deleteAllContainers()
+
+	logDone("create - portrange")
```

Comment:
No, it is just about rewording done message. `port range publishing` instead of `portrange`.